### ISSUE TITLE
Error in ZMQ subprocess: "Lisp nesting exceeds ‘max-lisp-eval-depth’"

### ISSUE THREAD

**Issue by antoniojkim** [nnicandro/emacs-jupyter#321]
Moved from Emacs 27.1 to Emacs 28.0.50 to try out native compilation. Encountered the following error message and a jupyter kernel crash:
```
error in process filter: zmq--subprocess-filter: Error in ZMQ subprocess: error, ("Lisp nesting exceeds ‘max-lisp-eval-depth’")
error in process filter: Error in ZMQ subprocess: error, ("Lisp nesting exceeds ‘max-lisp-eval-depth’")
```
Encountered when in an org file and executing any `jupyter-python` block, e.g.
```elisp
#+BEGIN_SRC jupyter-python
print("Hello World")
#+END_SRC
```
Attempting to execute block yields the above error message and then Emacs hangs while `Requesting kernel info...` until I manually quit using `C-g`.

Can someone please look into this. I will be using the suggestion in [this comment](https://github.com/nnicandro/emacs-jupyter/issues/297#issuecomment-719762173) in the meantime to disable native compiling of `emacs-jupyter`.

**Comment by antoniojkim** [comment]
I just came to realization that its perhaps not the native compilation thats the problem. I tried disabling native compilation for this package and have verified that it is not being native compiled and yet I still run into the same issue. So, I'm thinking it has something to do with Emacs 28.0.50 itself. Any ideas for a fix?

**Comment by nnicandro** [comment]
Looks like it could be the same issue that was solved by #320 which was merged into master just last night.  Maybe try updating emacs-jupyter?

**Comment by antoniojkim** [comment]
awesome! Updating emacs-jupyter fixed it. Thanks!
